### ISSUE TITLE
Broaden the search surface

### ISSUE THREAD

**Issue by pixelastic** [algolia/talksearch-scraper#43]
This can be found with `recognize who was`, but not with `recognize human`.

![image](https://user-images.githubusercontent.com/283419/41465184-4b44feec-709d-11e8-8f6f-072796e570f3.png)

The current implementation creates one record per line of transcript. As those keywords are on two different lines, they are not found. The solution might be to index more than one line in each record (line, line +1 and line -1), to broaden the search surface and fix this issue. 

We would still need to create one record per line (not one record per 3 lines), to keep the correct time to jump to.

**Comment by Haroenv** [comment]
🤔 what if we are okay with the lesser exact time? When I did the initial implementation I went -2s on the subtitle to give you a little time to adjust and pay attention for the subtitles before showing them. This would mean that the three lines wouldn't really be a problem

**Comment by pixelastic** [comment]
I'm not sure I understand what you're describing here. 

I closed the issue because I "fixed" it in that the caption content now contains two lines of caption instead of one, with some overlap. So record 1 will have line 1 and line 2, record 2 will have 2 and 3, etc.

I found that depending on the number of silences or _hmm_ in the sound, the caption start can sometimes be off by up to 2s even without adding a delay.
